docs/developer/release/release_utils.py

```
@@ -289,6 +289,9 @@ def update_repo(path: Path, bareflag: bool) -> None:
 
 def commit_tag_and_push(version: str, path: Path, tag_prefix: str) -> None:
     """Commit the changes made for this release, add a tag, and push these changes."""
+    # Remove the pre-commit hook because it can cause errors.
+    execute(f"rm .git/hooks/pre-commit", working_dir=path)
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)